crowbar_framework/app/assets/javascripts/application.js

```diff
@@ -256,4 +257,41 @@ jQuery(document).ready(function($) {
     },
     500
   );
+
+  // Backup and restore
+  $('[data-toggle="tooltip"]').tooltip({html : true});
+  $('[data-toggle="inline"]').popover({
+    html : true,
+    content: function() {
+      return $($(this).data('inline')).html();
+    }
+  });
+
+  $('[name="fileupload"]').fileinput({
```

Comment:
Please be more specific with the selector, a controller name is always included as a class on body element.